templates/taxonomy-topic.php

```diff
@@ -49,25 +53,37 @@
 					$related_faq_html .=  UCF_FAQ_Common::display_faq( $post, $show, 'h5' );
 				}
 
-				if( $related_faq_html ) {
+				if( $related_faq_html ) :
 				?>
-					<h2 class="<?php UCF_FAQ_Config::add_athena_attr( 'h3 mt-5 mb-4' ); ?>">Related FAQs</h2>
+					<h2 class="<?php UCF_FAQ_Config::add_athena_attr( 'h3 mt-5 mb-4' ); ?>">
+						<?php echo get_field( 'related-faq-title', get_queried_object() ); ?>
+					</h2>
 				<?php
 					echo $related_faq_html;
-				}
+				endif;
 
+				if( $cta_text = get_field( 'faq-topic-footer-cta-text', get_queried_object() ) ) :
+			?>
+					<a href="<?php echo site_url() . get_field( 'faq-topic-footer-cta-url', get_queried_object() ); ?>"
+						class="<?php UCF_FAQ_Config::add_athena_attr( 'btn btn-primary mt-4' ); ?>">
+						<?php echo $cta_text; ?>
+					</a>
+			<?php
+				endif;
 			?>
 				</div>
 
-				<div class="<?php UCF_FAQ_Config::add_athena_attr( 'col-md-4 mt-3' ); ?>">
-					<h2 class="<?php UCF_FAQ_Config::add_athena_attr( 'h5' ); ?>">TODO: Spotlight</h2>
-				</div>
+				<?php if( $spotlight ) : ?>
+					<div class="col-md-4 mt-3">
```

Comment:
We should add another margin class here for mobile layouts, currently the spotlight is sitting right underneath the 'View All FAQs' button. Using `mt-5 mt-md-3` adds more space in mobile and keeps the alignment with the first question in larger screen sizes